actors/builtin/power/power_state.go

```diff
@@ -41,6 +43,9 @@ type State struct {
 	// Cron will iterate every epoch between this and the current epoch inclusively to find tasks to execute.
 	FirstCronEpoch abi.ChainEpoch
 
+	// Last epoch power cron tick has been processed.
+	LastCronEpoch abi.ChainEpoch
```

Comment:
These two names follow an idiomatic pattern but mean very different things (or, from another point of view, the same thing two different ways).

Change either or both names so that they don't seem so related. Maybe `FirstUnprocessCronEpoch`? `LastProcessedCronEpoch`?